src/main/resources/init_db.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS books (
+    id BIGINT PRIMARY KEY NOT NULL AUTO_INCREMENT,
```

Comment:
Primary key constraint already contains not-null in it, so it's redundant
```suggestion
    id BIGINT PRIMARY KEY AUTO_INCREMENT,
```